wishlist.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="stylesheet" href="https://lucid-u.netlify.app/component/library.css">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="wishlist.css">
+    <title>LensMart | Wishlist</title>
+</head>
+<body>
+    <div class="header">
+        <a href="./index.html"><p class="brand heading3">LensMart</p></a>
+        <div class="header__search" >
+            <input type="text" name="" id="">
+            <span class="material-icons search__icon ">search</span>
+        </div>
+        <div class="header__icons" >
+            <a href="./login.html"><button class="btn btn-primary" >Login</button></a>
+            <a href="./wishlist.html">
+                <div class="relative">
+                <span class="material-icons">favorite_border</span>
+                <div class="badge badge-top-right badge-success">0</div>
+            </div>
+            </a>
+            <a href="./cart.html">
+                <div class="relative">
+                <span class="material-icons">shopping_cart</span>
+                <div class="badge badge-top-right badge-success">0</div>
+          </div>
+            </a>
+        </div>
+    </div>
+    <div class="wishlist__page">
+
+        <div class="wishlist__list" >
+            <div class="card relative wishlist__card " >
+            <img class="card-image" src="./assets/images/antiglare.jpg" alt="" />
```

Comment:
Write something meaningful in alt attribute.

Reply:
thanks will make the change